website/pages/chapters/assessing_performance.js

```diff
@@ -156,6 +195,11 @@ export default function AssessingPerformance() {
         </p>
 
         <p>
+          <MatthewComment>
+            This is a huge leap from the description of the joint distribution
+            above to this notation. I think more care is going to be needed to
+            confidently introducte the expected loss.
+          </MatthewComment>
```

Comment:
Yeah, there is an interesting tension here since this is essentially how we introduce it in class. I don't want to spend a ton of time setting up some notation that we don't emphasize, but do want to add it for clarity. 

I agree, this is one of the things that needs work!